debian/syslog-ng.systemd

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=System Logger Daemon
 Documentation=man:syslog-ng(8)
+Wants=network.target network-online.target
+After=network.target network-online.target
```

Comment:
Why it is required? I understand that with the journal, the importance of the local logging is decreased but honestly, I still check /var/log/messages first for every problem. And this would mean that in case of a network error, syslog-ng would not start.

Reply:
I don't know the specifics, this was added to usptream due to a bug report where syslog-ng would fail to start when bound to an interface, but I'll check that out in details.

Reply:
Here's a link to the discussion:

https://github.com/syslog-ng/syslog-ng/issues/2667

All in all we are matching rsyslog on redhat here, which depends on the same. 

Note: neither of these options mean that syslog-ng will fail to start if network is unavailable. Only that an attempt is made to start those before syslog-ng.